### ISSUE TITLE
Remove exteranally hosted JS for Monaco Editor

### ISSUE THREAD

**Issue by johnaohara** [Hyperfoil/Horreum#364]
**Describe the bug**

Externally hosted JS are key components of UI. 

For example, the JSON and code editor is based on Monaco Editor (https://github.com/Microsoft/monaco-editor) but the JS for the editor is linked to a remote host: https://cdn.jsdelivr.net/npm/monaco-editor@0.28.1/min/vs/editor/editor.main.js

All JS should be served from the Horreum webapp and not rely on externally hosted JS for functionality.  

Not only is there a security concern, but the Horreum UI will not function as intended in air-gapped envs or when https://cdn.jsdelivr.net fails to serve the required JS files

**Comment by johnaohara** [comment]
From an intial scan, it looks like monaco is the only affected library, but we should audit all the deps to identify the ones that load external resources at runtime

**Comment by johnaohara** [comment]
From an intial scan, it looks like monaco is the only affected library, but we should audit all the deps to identify the ones that load external resources at runtime

**Comment by jesperpedersen** [comment]
Yeah, we probably need a support matrix for the Monaco Editor stuff with the React dependencies as well.

We have about 1.5y worth of updates within this area.
